SurveyApp/Extensions/UIKit/UIView/UIView+Present.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UIView+Present.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 7/11/21.
+//
+
+import UIKit
+
+extension UIView {
+    
+    func presentViewOnTopWindow(_ duration: TimeInterval = 0.3, for dismissAfter: TimeInterval = 0, _ completion: EmptyCompletion? = nil) {
+        alpha = 0
+        frame = UIScreen.main.bounds
+        let windows = UIApplication.shared.windows.first { $0.isKeyWindow }
+        windows?.addSubview(self)
```

Comment:
Stop everything if `window` is not available
```suggestion
        guard let window = UIApplication.shared.windows.first ({ $0.isKeyWindow }) else { return }
        alpha = 0
        frame = UIScreen.main.bounds
        window.addSubview(self)
```